src/server/sql/group/create_groups_tables.sql

```diff
@@ -4,7 +4,7 @@
 
 CREATE TABLE groups (
 	id SERIAL PRIMARY KEY NOT NULL,
-	name VARCHAR(50) UNIQUE NOT NULL
+	name VARCHAR(50) UNIQUE NOT NULL CHECK (char_length(name) >= 1)
```

Comment:
This seems useful, but shouldn't there also be a server-side check in routes/groups.js that checks the name length (and sanitizes other input such as group and meter ids) and returns the correct status code to the client?

Reply:
This is enough to reject empty group names. I could add a second check, but it would be functionally equivalent. 
At the moment I'm trusting that the provided group and meter IDs are valid. I'm not sure, but I think the queries to add children should fail if the given IDs are invalid.